activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/TasksIT.java

```diff
@@ -279,6 +279,21 @@ public void shouldBeAbleToDeleteTask() {
         assertThat(delete.getStatusCodeValue()).isEqualTo(HttpStatus.OK.value());
     }
 
+    @Test
+    public void adminShouldBeAbleToDeleteTask() {
+        //given
+        CloudTask standaloneTask = taskRestTemplate.createTask(TaskPayloadBuilder.create().withName("parent task").withDescription("This is my parent task").build());
+        //when
+        keycloakSecurityContextClientRequestInterceptor.setKeycloakTestUser("testadmin");
+        ResponseEntity<CloudTask> delete = taskRestTemplate.adminDelete(standaloneTask);
+
+        //then
+        assertThat(delete.getStatusCodeValue()).isEqualTo(HttpStatus.OK.value());
+        
+        //restore user
+        keycloakSecurityContextClientRequestInterceptor.setKeycloakTestUser("hruser");
```

Comment:
Small detail. There is not need to restore `hruser` here because it's already been set in `setUp` method. So it will be automatically set to `hruser` before starting every test. I'm removing it and merge this pr.